polaris.shopify.com/src/components/HomePage/HomePage.module.scss

```diff
@@ -196,4 +196,16 @@
       mix-blend-mode: plus-lighter;
     }
   }
+
+  .Poster {
```

Comment:
This was being used on the HomePage post rebase, (also in main) but not specified in the `scss` file 
Added it back in

Reply:
--Update-- 
I'm getting cryptic errors around conflicting paths likely to do with how `[...slug].tsx` works atm as well as bunch of other build failures. I'd like to resolve this in a separate PR as this may be a more involved change to sandbox routes.